src/sections/reference.html

```diff
@@ -0,0 +1,41 @@
+<div class="relative">
+  <img
+    class="h-sm-reference md:h-md-reference lg:h-lg-reference"
+    src="/img/Rectangle-3.webp"
+    alt="a yellow rectangle"
+  />
+  <div class="flex justify-end items-end">
+    <ul
+      class="font-body text-right px-8 text-xl absolute top-0 pt-20 gap-3 w-w-46-reference md:w-w-reference md:right-24 md:bottom-28 md:text-2xl lg:bottom-36 lg:text-2xl lg:right-24">
```

Comment:
- The lg:text-2xl is not needed since you have the same class on md:text-2xl.
- Font-body is not a good naming since you can rename the fonts, just rename it as Outfit and add its value. 
- Please avoid using tailwind classes that are not defined already in tailwind unless it's necessary. I.e those vh remove them and use the height and width that already exist in tailwind classes.